wire-ios-sync-engine/Tests/Source/Integration/ConversationTests+ClearingHistory.swift

```diff
@@ -94,12 +94,12 @@ class ConversationTests_ClearingHistory: ConversationTestsBase {
         XCTAssertTrue(objectIDs!.contains(conversationID!))
     }
 
-    func testThatDeletedConversationsStayDeletedAfterResyncing() {
+    func testThatDeletedConversationsStayDeletedAfterResyncing() throws {
         // given
         let messagesCount: UInt = 5
         loginAndFillConversationWithMessages(mockConversation: self.groupConversation, messagesCount: messagesCount)
 
-        var conversation = self.conversation(for: self.groupConversation!)
+        var conversation = conversation(for: self.groupConversation!)
         XCTAssertEqual(conversation!.allMessages.count, 5)
```

Comment:
`var conversation = try XCTUnwrap(conversation(for: self.groupConversation!))` would help to avoid force unwrap `conversation` later in this test.

Reply:
done